assets/javascripts/tests.js

```diff
@@ -502,4 +488,41 @@ function setupLazyLoadingFailedSteps() {
             this.hasFailedSteps = false;
         });
     });
+}
+
+function showJobDependency(deps) {
+    var parents = deps.parents;
+    var children = deps.children;
+    var depsTooltip = [];
+    var result = {};
+    depsTooltip.quantify = function(quandity, singular, plural) {
```

Comment:
I think you mean "quantity"

Reply:
fixed. I did not notice it, just copy and paste. Thanks.